flux_led/__init__.py

```diff
@@ -622,9 +635,9 @@ def setRgb(self, r,g,b, persist=True):
             msg = bytearray([0x31])
         else:
             msg = bytearray([0x41])
-        msg.append(r)
-        msg.append(g)
-        msg.append(b)
+        msg.append(int(r))
```

Comment:
I added getRgbw(). In  my setup, both functions work fine with both RGB and RGBW lights. Setting the white components does nothing and when getting it 0 is returned.